h/streamer/metrics.py

```diff
@@ -40,13 +42,13 @@ def websocket_metrics(queue):
         yield f"{PREFIX}/Worker/Pool/Used", pool.size - free
 
 
-NEW_RELIC_CONFIG_REF = importlib_resources.files("h.streamer") / "conf/newrelic.ini"
+NEW_RELIC_CONFIG_PATH = Path("conf/websocket-newrelic.ini")
```

Comment:
We actually don't need to change this file at all. We could actually leave this file in its original location at `h/streamer/conf/newrelic.ini` if we'd like to continue using `importlib_resources` to locate it. But I thought it might be nice to keep all the config files together on the root `conf/` directory like we usually do.